konfera/views.py

```diff
@@ -60,3 +65,25 @@ def event_details_view(request, event_slug):
     return render(request=request,
                   template_name='konfera/event_details.html',
                   context=context, )
+
+
+def cfp_form_view(request, event_slug):
+    speaker_form = SpeakerForm(request.POST or None, prefix='speaker')
+    talk_form = TalkForm(request.POST or None, prefix='talk')
+    if speaker_form.is_valid() and talk_form.is_valid():
+        speaker_instance = speaker_form.save()
+        talk_instance = talk_form.save(commit=False)
+        talk_instance.primary_speaker = speaker_instance
+        talk_instance.status = CFP
+        talk_instance.event = Event.objects.get(slug=event_slug)
+        talk_instance.save()
+        message_text = _("Your talk proposal successfully created")
```

Comment:
@TomasHo @ricco386 I just realized. Shouldn't be here `ugettext` instead of `ugettext_lazy` ?